.circleci/config.yml

```diff
@@ -833,11 +847,19 @@ workflows:
       - cwag-precommit
       - cwf-integ-test:
           <<: *only_master
+      - xwfm-test:
+          <<: *only_master
       - cwag-deploy:
           <<: *only_master
           requires:
             - cwag-precommit
             - cwf-integ-test
+      - xwfm-deploy:
+          <<: *only_master
+          requires:
+            - xwfm-test
+            - cwag-deploy
```

Comment:
One other question: Based on our earlier discussion, do you think it makes sense to isolate the xwf-m stuff in a separate workflow that runs on a cron? Since xwfm-test doesn't block cwag-deploy, we could finish this workflow and ship only the cwag artifacts.